config/names/death_commando.txt

```diff
@@ -63,7 +63,7 @@ Splint Chesthair
 Stabby McGee
 Stump Beefgnaw
 Stump Chunkman
-THAT DAMN FAGGOT TRAITOR GEORGE MELONS
+THAT DAMN FUCKING TRAITOR GEORGE MELONS
```

Comment:
This one's fucking instead of the CANDY_ASS define.

Reply:
that's right, because there's no strings in sight
i would fix the name whenever these get loaded, but they never do - we don't use these names for deathsquaddies